check_sites.py

```diff
@@ -145,12 +146,14 @@ def main():
         except Exception as inst:
             error_texts.append(inst)
     # This message allows us to check the succes of our action
-    if rws_checker.error_list or error_texts:
+    if rws_checker.error_list or error_texts or rws_checker.warning_texts:
         for checker_error in rws_checker.error_list:
             print(checker_error)
         for error_text in error_texts:
             print(error_text)
-    else:
+        for warning in rws_checker.warning_texts:
```

Comment:
Maybe add a comment here that indicates it's important for the warnings to be *after* errors? Unless you choose to remove that limitation, per my other comment.